presidio-image-redactor/Pipfile

```diff
@@ -9,9 +9,15 @@ flask = "==1.1.2"
 pytesseract = "==0.3.7"
 presidio-analyzer = ">=1.9.0"
 pillow = "==8.1.1"
+importlib-metadata = { version = "==3.7.3", markers = "python_version<'3.8'" }
+typing-extensions = { version = "==3.7.4.3", markers = "python_version<'3.8'" }
+zipp = { version = "==3.4.1", markers = "python_version>='3.6'" }
 
 [dev-packages]
 pytest = "*"
 flake8 = { version = "==3.7.9" }
 pep8-naming = "*"
-flake8-docstrings = "*"
\ No newline at end of file
+flake8-docstrings = "*"
+importlib-metadata = { version = "==3.7.3", markers = "python_version<'3.8'" }
+typing-extensions = { version = "==3.7.4.3", markers = "python_version<'3.8'" }
+zipp = { version = "==3.4.1", markers = "python_version>='3.6'" }
```

Comment:
Should these be both on dev and regular?

Reply:
Validating it...

Reply:
Removed.